content/sklearn/concepts/biclustering/biclustering.md

```diff
@@ -0,0 +1,111 @@
+---
+Title: 'Biclustering'
+Description: 'A technique for grouping rows and columns of a matrix to discover local patterns in data.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+  - 'Machine Learning'
+Tags:
+  - 'Clustering'
```

Comment:
'Clustering' is not a valid tag. Refer to [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) for reference.

```suggestion
```